extras.go

```diff
@@ -74,6 +75,38 @@ func addExtrasRoutes(r chi.Router) {
 		format.JSON(w, 200, Response{ID: id})
 	})
 
+	r.Post("/tags", func(w http.ResponseWriter, r *http.Request) {
+		r.ParseForm()
+		name := r.Form.Get("name")
+		color := r.Form.Get("color")
+		value := strings.ReplaceAll(name, " ", "")
+
+		res, _ := conn.Exec("INSERT INTO tag (name, value, color) VALUES (?, ?, ?)", name, value, color)
+
+		cid, _ := res.LastInsertId()
+		format.JSON(w, 200, Response{ID: strconv.FormatInt(cid, 10), Value: value})
+	})
+
+	r.Put("/tags/{id}", func(w http.ResponseWriter, r *http.Request) {
+		r.ParseForm()
+		id := chi.URLParam(r, "id")
+		name := r.Form.Get("name")
+		color := r.Form.Get("color")
+		value := strings.ReplaceAll(name, " ", "")
+
+		conn.Exec("UPDATE tag SET name = ?, value = ?, color = ? WHERE id = ?", name, value, color, id)
+		format.JSON(w, 200, Response{ID: id})
```

Comment:
not quite sure, but why we are returning value during adding, but not returning after updating?

Reply:
both add and update methods return full TagInfo now